src/Symfony/Bundle/FrameworkBundle/CacheWarmer/RouterCacheWarmer.php

```diff
@@ -22,16 +23,18 @@
  */
 class RouterCacheWarmer implements CacheWarmerInterface
 {
+    private $container;
     protected $router;
```

Comment:
Let's revert changes on this file, it's a BC break because of this protected property.